browser/scss/item_page/item_page.scss

```diff
@@ -39,6 +30,31 @@
 	}
 }
 
+.product_panel {
```

Comment:
i think if we use bootstrap panel elements we won't have to play and massage the paddings and styles as much. No change necessary here.